bundle/CMakeLists.txt

```diff
@@ -123,7 +123,12 @@ endif()
    endif()
 
   endif(CLONE_JCSDADATA)
+     set(RDAS_RRFS_DATA_ROOT "$ENV{RDASAPP_RRFS_TESTDATA}/jcsda")
+     set(RDAS_RRFS_DATA_ROOT "/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/rrfs-test-data") #clttodo
```

Comment:
Don't set this here, this is hard coded for Hera, so it should not be in the bundle CMakeLists.txt. Instead, define this data root in the modulefile?

Reply:
Also the default data directory is being overwritten with this path?

Reply:
@CoryMartin-NOAA  Yes. the RDAS_RRFS_DATA_ROOT is supposed to be defined outside of RDASapp itself while being defined through env variables as the line 126 indicated.   The current Line 127 is a temporary setup and I will see if there could be a location on hera /orion shared by EMC RDAS developers.